index.bs

```diff
@@ -1406,10 +1439,14 @@ Data truncation will occur when the specified value exceeds the range of the spe
   </summary>
     1. If [=this=].{{MLGraphBuilder/[[hasBuilt]]}} is true, then [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
     1. Set |value| to the result of [=casting=] |value| to |type|.
+    1. Let |bytes| be a [=byte sequence=] represented by |value|.
+
+      ISSUE(TODO): Specify this in terms of https://tc39.es/ecma262/multipage/structured-data.html#sec-numerictorawbytes.
```

Comment:
nit: Make this a link (bikeshed didn't do it automagically?!?)

Reply:
If we want to keep this (see previous note about bytes), I think referencing the ECMAScript algorithm is fine. Just write `[$NumericToRawBytes$](|type|, |value|, true)` - it's true that the types don't quite align between the algorithms but it's really really really obvious that "int8" in our enum is the same as INT8 in https://tc39.es/ecma262/multipage/indexed-collections.html#sec-typedarray-objects

Reply:
Used `[$NumericToRawBytes$]` in the latest patch to show what that looks like, but this could soon be moot